frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-row.tsx

```diff
@@ -146,20 +150,27 @@ export type VMDiskRowProps = {
 };
 
 export const DiskRow: React.FC<VMDiskRowProps> = ({
-  obj: { disk, ...restData },
+  obj: { disk, diskValidations, templateValidations, ...restData },
   customData: { isDisabled, withProgress, vmLikeEntity, columnClasses },
   index,
   style,
 }) => {
   return (
     <DiskSimpleRow
       data={restData}
+      validation={
+        diskValidations && {
+          name: diskValidations.validations.name,
+          size: diskValidations.validations.size,
+          diskInterface: diskValidations.validations.diskInterface,
```

Comment:
lets show the other validations in source

Reply:
there is a `source` validation field

Reply:
not on `UIDiskValidation` type.
Here I'm using the result from `validateDisk` which returns `UIDiskValidation` object and not `StorageSimpleDataValidation`

Reply:
+1 for fixing the source mapping